f8a_report/helpers/sentry_report_helper.py

```diff
@@ -138,3 +138,11 @@ def retrieve_events(self, issue_id):
         except NameError as e:
             logger.error('Name not found while parsing. Reason: %r' % e)
         return events
+
+
+def generate_sentry_report(start_date: str, end_date: str):
```

Comment:
This is Entry Point for Sentry Reporting.

Reply:
What about the logic of collecting the sentry error report which is currently in the code? where was it getting called from earlier?

Reply:
@yzainee here, along with v1 reporting https://github.com/fabric8-analytics/f8a-stacks-report/blob/master/f8a_report/helpers/report_helper.py#L868

Reply:
strange logic existed.. im fine with the changes now.